R/transformCounts.R

```diff
@@ -24,12 +24,22 @@
 #' @param name \code{Character scalar}. A name for the column of the 
 #'   \code{colData} where results will be stored. (Default: \code{"method"})
 #'
-#' @param ... additional arguments passed on to \code{vegan:decostand}:
+#' @param ... additional arguments.
 #' \itemize{
-#'   \item \code{reference}: \code{Character scalar}. use to
+#'   \item \code{reference}: \code{Character scalar}. Used to
 #'   to fill reference sample's column in returned assay when calculating alr. 
 #'   (Default: \code{NA})
 #'   \item \code{ref_vals} Deprecated. Use \code{reference} instead.
+#'   \item \code{percentile}: \code{Numeric scalar}.  Used to set the 
```

Comment:
Consider \code{percentile} \code{Numeric scalaer} (css)

--> It might be easier for user to see where this applies

Reply:
ACK.